service/core/serverObj/v2ray.go

```diff
@@ -228,21 +234,35 @@ func (v *V2Ray) Configuration(info PriorInfo) (c Configuration, err error) {
 				},
 			}
 		case "vless":
+			security := v.Security
+			if security == "" {
+				security = "auto"
+			}
+			flow := v.Flow
+			if flow == "" {
+				flow = "xtls-rprx-direct"
+			}
 			core.Settings.Vnext = []coreObj.Vnext{
 				{
 					Address: v.Add,
 					Port:    port,
 					Users: []coreObj.User{
 						{
-							ID:         id,
 							Encryption: "none",
+							Flow:       flow,
+							ID:         id,
+							Level:      8,
```

Comment:
Why hardcode?

Reply:
Due to some issues like this:
https://github.com/v2ray/v2ray-core/issues/569

Some users may experience slow connection on the Policy Level of user so we can have set it static to prevent any restriction.

If you want me add an extra field for it on settings, please let me know.

Reply:
Though there is some feedback about this, the issue you mention here is too old.

However, set it 8 is not a bad thing.